### PR TITLE
(feature) Make it possible to install recipes magazin in the installer

### PR DESCRIPTION
**Pull request by dawehner**
* Is this a good idea
* Is this simple approach the best idea?

### CODE REVIEW

File: contenta_jsonapi.profile [hunk 1]
```diff
@@ -15,4 +15,27 @@ use Drupal\Core\Form\FormStateInterface;
...
   $form['site_information']['site_name']['#placeholder'] = t('Contenta JSON API');
+
+  $form['contenta_jsonapi'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Contenta settings'),
+    '#weight' => -10,
+  ];
+    
+  $form['contenta_jsonapi']['include_recipes_magazin'] = [
+    '#title' => t('Include the recipes magazin'),
```
Comment (e0ipso) [inline contenta_jsonapi.profile]: `'Install the demo content'`.

And then add a `#description`: `'By installing the demo content Contenta will create a set of content types and populate them with real data. You can remove the demo content and the associated content types with a single click whenever you want.'`.

Comment (dawehner) [inline contenta_jsonapi.profile]: > 'Install the demo content'.

+1 to not couple to the concrete default content. People don't have to know about it.

Comment (dawehner) [inline contenta_jsonapi.profile]: We are using your suggested text now. Switched the checkbox on by default

File: contenta_jsonapi.profile [hunk 2]
```diff
@@ -15,4 +15,29 @@ use Drupal\Core\Form\FormStateInterface;
...
+  $form['#submit'][] = 'contenta_jsonapi_install_configure_form_submit';
+}
+
+/**
+ * Submit handler for install_configure_form().
+ */
+function contenta_jsonapi_install_configure_form_submit(&$form, FormStateInterface $form_state) {
+  if ($form_state->getValue('include_recipes_magazin')) {
+    drupal_set_message(t('Recipe magazin installed'));
+    \Drupal::service('module_installer')->install(['recipes_magazin']);
```
Comment (e0ipso) [inline contenta_jsonapi.profile]: I'm going to add a follow up to catch the installation response for the message.

Comment (e0ipso) [inline contenta_jsonapi.profile]: See 0fdba7f5b1a5fa144d34a8de534add18de01083a

**Pull request merged**